source/connect/tls.txt

```diff
@@ -265,4 +265,4 @@ API Documentation
 To learn more about configuring TLS for the {+driver-short+},
 see the following API documentation:
 
-- :ref:`MongoDB\Client <php-mongodb-client>`
\ No newline at end of file
+- :ref:`MongoDB\Client <php-api-mongodbclient>`
```

Comment:
Why does this used a named reference instead of `:phpclass:` or `:phpmethod:`? Those are used throughout existing docs (e.g. [GridFS tutorial](https://github.com/mongodb/docs-php-library/blob/master/source/tutorial/gridfs.txt)).

Reply:
I didn't know about these directives (and apparently the other writers didn't either). We've agreed to use them for API links.